sql/src/main/java/io/crate/metadata/settings/session/SessionSetting.java

```diff
@@ -61,6 +66,23 @@
         this.type = type;
     }
 
+    public void apply(Functions functions,
+                      Row parameters,
+                      CoordinatorTxnCtx txnCtx,
+                      List<Symbol> symbols,
+                      SessionContext sessionContext,
+                      SubQueryResults subQueryValues
```

Comment:
can we pass in a mapper function instead? e.g. `Function<? super Symbol, Object> valuesMapper`